src/fe-gtk/palette.c

```diff
@@ -37,135 +37,126 @@
-	{0, 0x2e2e, 0x0000, 0x4747}, /* 25 */
-	{0, 0x4747, 0x0000, 0x4747}, /* 26 */
-	{0, 0x4747, 0x0000, 0x2a2a}, /* 27 */
-	{0, 0x7474, 0x0000, 0x0000}, /* 28 */
-	{0, 0x7474, 0x3a3a, 0x0000}, /* 29 */
-	{0, 0x7474, 0x7474, 0x0000}, /* 30 */
-	{0, 0x5151, 0x7474, 0x0000}, /* 31 */
-	{0, 0x0000, 0x7474, 0x0000}, /* 32 */
-	{0, 0x0000, 0x7474, 0x4949}, /* 33 */
-	{0, 0x0000, 0x7474, 0x7474}, /* 34 */
-	{0, 0x0000, 0x4040, 0x7474}, /* 35 */
-	{0, 0x0000, 0x0000, 0x7474}, /* 36 */
-	{0, 0x4b4b, 0x0000, 0x7474}, /* 37 */
-	{0, 0x7474, 0x0000, 0x7474}, /* 38 */
-	{0, 0x7474, 0x0000, 0x4545}, /* 39 */
-	{0, 0xb5b5, 0x0000, 0x0000}, /* 40 */
-	{0, 0xb5b5, 0x6363, 0x0000}, /* 41 */
-	{0, 0xb5b5, 0xb5b5, 0x0000}, /* 42 */
-	{0, 0x7d7d, 0xb5b5, 0x0000}, /* 43 */
-	{0, 0x0000, 0xb5b5, 0x0000}, /* 44 */
-	{0, 0x0000, 0xb5b5, 0x7171}, /* 45 */
-	{0, 0x0000, 0xb5b5, 0xb5b5}, /* 46 */
-	{0, 0x0000, 0x6363, 0xb5b5}, /* 47 */
-	{0, 0x0000, 0x0000, 0xb5b5}, /* 48 */
-	{0, 0x7575, 0x0000, 0xb5b5}, /* 49 */
-	{0, 0xb5b5, 0x0000, 0xb5b5}, /* 50 */
-	{0, 0xb5b5, 0x0000, 0x6b6b}, /* 51 */
-	{0, 0xffff, 0x0000, 0x0000}, /* 52 */
-	{0, 0xffff, 0x8c8c, 0x0000}, /* 53 */
-	{0, 0xffff, 0xffff, 0x0000}, /* 54 */
-	{0, 0xb2b2, 0xffff, 0x0000}, /* 55 */
-	{0, 0x0000, 0xffff, 0x0000}, /* 56 */
-	{0, 0x0000, 0xffff, 0xa0a0}, /* 57 */
-	{0, 0x0000, 0xffff, 0xffff}, /* 58 */
-	{0, 0x0000, 0x8c8c, 0xffff}, /* 59 */
-	{0, 0x0000, 0x0000, 0xffff}, /* 60 */
-	{0, 0xa5a5, 0x0000, 0xffff}, /* 61 */
-	{0, 0xffff, 0x0000, 0xffff}, /* 62 */
-	{0, 0xffff, 0x0000, 0x9898}, /* 63 */
-	{0, 0xffff, 0x5959, 0x5959}, /* 64 */
-	{0, 0xffff, 0xb4b4, 0x5959}, /* 65 */
-	{0, 0xffff, 0xffff, 0x7171}, /* 66 */
-	{0, 0xcfcf, 0xffff, 0x6060}, /* 67 */
-	{0, 0x6f6f, 0xffff, 0x6f6f}, /* 68 */
-	{0, 0x6565, 0xffff, 0xc9c9}, /* 69 */
-	{0, 0x6d6d, 0xffff, 0xffff}, /* 70 */
-	{0, 0x5959, 0xb4b4, 0xffff}, /* 71 */
-	{0, 0x5959, 0x5959, 0xffff}, /* 72 */
-	{0, 0xc4c4, 0x5959, 0xffff}, /* 73 */
-	{0, 0xffff, 0x6666, 0xffff}, /* 74 */
-	{0, 0xffff, 0x5959, 0xbcbc}, /* 75 */
-	{0, 0xffff, 0x9c9c, 0x9c9c}, /* 76 */
-	{0, 0xffff, 0xd3d3, 0x9c9c}, /* 77 */
-	{0, 0xffff, 0xffff, 0x9c9c}, /* 78 */
-	{0, 0xe2e2, 0xffff, 0x9c9c}, /* 79 */
-	{0, 0x9c9c, 0xffff, 0x9c9c}, /* 80 */
-	{0, 0x9c9c, 0xffff, 0xdbdb}, /* 81 */
-	{0, 0x9c9c, 0xffff, 0xffff}, /* 82 */
-	{0, 0x9c9c, 0xd3d3, 0xffff}, /* 83 */
-	{0, 0x9c9c, 0x9c9c, 0xffff}, /* 84 */
-	{0, 0xdcdc, 0x9c9c, 0xffff}, /* 85 */
-	{0, 0xffff, 0x9c9c, 0xffff}, /* 86 */
-	{0, 0xffff, 0x9494, 0xd3d3}, /* 87 */
-	{0, 0x0000, 0x0000, 0x0000}, /* 88 */
-	{0, 0x1313, 0x1313, 0x1313}, /* 89 */
-	{0, 0x2828, 0x2828, 0x2828}, /* 90 */
-	{0, 0x3636, 0x3636, 0x3636}, /* 91 */
-	{0, 0x4d4d, 0x4d4d, 0x4d4d}, /* 92 */
-	{0, 0x6565, 0x6565, 0x6565}, /* 93 */
-	{0, 0x8181, 0x8181, 0x8181}, /* 94 */
-	{0, 0x9f9f, 0x9f9f, 0x9f9f}, /* 95 */
-	{0, 0xbcbc, 0xbcbc, 0xbcbc}, /* 96 */
-	{0, 0xe2e2, 0xe2e2, 0xe2e2}, /* 97 */
-	{0, 0xffff, 0xffff, 0xffff}, /* 98 */
+	{1.0, 1.0, 1.0, 1.0}, /* 0 white */
```

Comment:
ideally this should be like (((float)0xabcdef) / 65535.0); the definiition is based on hex codes iirc